reference/constraints/Timezone.rst

```diff
@@ -0,0 +1,127 @@
+Timezone
+========
+
+Validates that a value is a valid timezone identifier (ie. ``Europe/Paris``).
```

Comment:
Kind of nitpicky, but it should be e.g., not ie.

see for example https://www.grammarly.com/blog/know-your-latin-i-e-vs-e-g/

> E.g. stands for exempli gratia and means “for example.” I.e. is the abbreviation for id est and means “in other words.”